app/views/case_details_offence.html

```diff
@@ -13,22 +13,22 @@
 {% endblock %}
 
 {% block content %}
+  {% if date_stamp %}
+    <div class="govuk-notification-banner govuk-notification-banner--success" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+      <div class="govuk-notification-banner__header">
+        <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+          Success
+        </h2>
+      </div>
+      <div class="govuk-notification-banner__content">
+        <h3 class="govuk-notification-banner__heading">
+          Your application has been date stamped
+        </h3>
+      </div>
+    </div>
+  {% endif %}
```

Comment:
We could move the date stamp snippet into an 'include' that can then be called into the various pages it's needed? Eg see /views/includes/head.html which is called into layout.html.